pkg/azbastion/bastion.go

```diff
@@ -104,11 +105,12 @@ func (b *Bastion) NewTunnelSession(targetHost string, port uint16) (*TunnelSessi
 		}
 	}
 
-	s, err := b.newSessionToken(targetHost, port)
+	s, err := b.newSessionToken(targetHost, port, scope)
 	if err != nil {
 		return nil, err
 	}
 
+	fmt.Printf("About to ")
```

Comment:
log?

Reply:
Removed. Forgot to delete debug statement